### PR TITLE
Converts main.m and the installer into Swift

### PR DESCRIPTION
**Pull request by zonble**
Also added the Credits.rtf file to show the links to the user manual and FAQ in the about window.

### CODE REVIEW

File: Source/Installer/BundleTranslocate.m [hunk 1]
```diff
@@ -0,0 +1,30 @@
+#import "BundleTranslocate.h"
```
Comment (lukhnos) [inline Source/Installer/BundleTranslocate.m:1]: Use the same copyright header from `BundleTranslocate.h`?

File: Source/main.swift [hunk 2]
```diff
@@ -0,0 +1,109 @@
+//
```
Comment (lukhnos) [inline Source/main.swift]: Let's use the new copyright header such as the one in `AppDelegate.swift` in this PR.

**Review by lukhnos — APPROVED** [review]
LGTM. Thanks!

**Comment by ShikiSuen** [comment]
NotoSansCJK is not shipped in macOS by default. What about changing the RTF font to systemfont or "Helvetica Neue" or "SF Pro Display (install separately but macOS can parse it to system font)" in the macOS TextEdit app if possible?

**Pull request merged**